portfolio/src/main/webapp/index.html

```diff
@@ -9,24 +9,26 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>Welcome to the porfolio of Lorna Armstrong.</p>
-
-      <h3>About Me</h3>
+      <h4>About Me</h4>
+      <img src= "/images/profile.jpg" alt="Profile Picture" width=200px height=200px margin-right=15px>    
       <p>My name is Lorna Armstrong and I am from Middlesbrough, England. 
-          I am a Computer Science student at the University of Edinburgh, 
-          heading into my third year in September 2020. Currently, I am 
-          working as a STEP Intern for Google, working remotely from Edinburgh.
-          This portfolio will show you a little bit about me.</p>
-
-      <img src= "/images/profile.jpg" alt="Profile Picture" style="width:200px;height:200px;">    
-      <img src= "/images/edinburgh.jpg" alt="Skyline of Edinburgh" style="width:400px;height:250px;">
-      <img src= "/images/skye.jpg" alt="My Border Collie, Skye" style="width:300px;height=300px;">
-      
+         I am a Computer Science student at the University of Edinburgh, 
+         heading into my third year in September 2020. Currently, I am 
+         working as a STEP Intern for Google, working remotely from Edinburgh.
+         This portfolio will show you a little bit about me.
+      </p>    
       <!-- Code from Original Template -->
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
-
+      <!-- This is the links section -->
+      <h3>
```

Comment:
usually the `h1..6` tags are used to define headings. [Ref](https://www.w3schools.com/tags/tag_hn.asp)
You can use [div](https://www.w3schools.com/tags/tag_div.asp) here instead to style the whole section the way you like :)

Reply:
Thanks for the note - that's really helpful!